tests/test_frozenlist.py

```diff
@@ -372,6 +387,23 @@ def test_deepcopy_multiple_references(self) -> None:
         assert len(copied[1]) == 3  # Should see the change
         assert len(shared) == 2  # Original unchanged
 
+    @pytest.mark.parametrize("freeze", [True, False])
```

Comment:
Could you also add human-readable IDs for context? This will make the test report easier to follow.